dxr/templates/nunjucks/results_container.html

```diff
@@ -1,10 +1,26 @@
 {% from "results.html" import results_list -%}
 {% from "results_header.html" import results_header -%}
+{% from "promoted.html" import promoted_list, promoted_header -%}
+{% from "switch_tree.html" import tree_menu -%}
 
-{{ results_header(result_count, result_count_formatted, top_of_tree, tree_tuples, tree) }}
+<div class="results-row">
 
-{% if results|length > 0 %}
-  <div class="results">
-    {{ results_list(results, www_root, tree) }}
+  <div class="result_header">
+  {{ results_header(result_count, result_count_formatted, top_of_tree, tree) }}
+  {{ tree_menu(tree_tuples, tree) }}
```

Comment:
Just give this a moment of thought to decide whether we still need to move this, since the promoted results are no longer on the right.